ui/package.json

```diff
@@ -41,6 +42,7 @@
         "ts-jest": "~29.1.1",
         "ts-loader": "~9.5.1",
         "ts-node": "~10.9.1",
+        "typed-scss-modules": "^8.0.0",
```

Comment:
Tämä lisää kyllä aika paljon oheispackageja (package-lock >+2.5k). Toimii varsin yksinkertaisesti tähän tarkoitukseen. 

Tuota aiempaa tyyppigenerointia (@teamsupercell/typings-for-css-modules-loader) käytetään edelleen, sillä se taas ei aiheuta yhtä laajojen muutosten triggeröintiä ajettaessa dev-palvelinta, jolloin webpack uudelleenbuildaa asioita nopeammin. Tästä on varmaan mahdollista päästä eroon tutkimalla asiaa vielä enemmän, tuo --updateStaleOnly nopeutti jo aika merkittävästi, muttei se ole silti yhtä nopea kuin tämä aiempi noissa osittaisissa uudelleenbuildeissa.

Reply:
Lisenssit onneksi validoidaan buildissakin joten sieltä ei pitäisi tulla yllätyksiä. Aika rajultahan tuo määrä kuulostaa. Voisi vaikka kurkata dailyssä että miltä tuo nyt sitten vaikuttaa sen nopeuden suhteen.